### ISSUE TITLE
[Buoyancy] Something wrong in mdpa or json

### ISSUE THREAD

**Issue by jginternational** [KratosMultiphysics/GiDInterface#443]
There is a problem in the execution:

This is seen in the output: 
`Model: trying to add to the Model two DISTINCT model parts with the same name. This should be possible (and it will be in the future) if they belong to two different root model_parts, but it is currently disallowed`

This is seen as an error:
```
Traceback (most recent call last):
  File "MainKratos.py", line 37, in <module>
    simulation.Run()
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\kratos\python_scripts\analysis_stage.py", line 47, in Run
    self.Initialize()
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\kratos\python_scripts\analysis_stage.py", line 70, in Initialize
    self._GetSolver().PrepareModelPart()
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\applications\convection_diffusion_application\python_scripts\coupled_fluid_thermal_solver.py", line 96, in PrepareModelPart
    self.thermal_solver.PrepareModelPart()
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\applications\convection_diffusion_application\python_scripts\convection_diffusion_base_solver.py", line 275, in PrepareModelPart
    KratosMultiphysics.TetrahedralMeshOrientationCheck(self.main_model_part, throw_errors).Execute()
RuntimeError: Error: Found a condition without any corresponding element. ID of condition = 57


in kratos/processes/tetrahedral_mesh_orientation_check.h:296:TetrahedralMeshOrientationCheck::Execute
   kratos/processes/tetrahedral_mesh_orientation_check.h:321:TetrahedralMeshOrientationCheck::Execute

```

**Comment by rubenzorrilla** [comment]
I came across with this during the development of the solver but it should be working right now... I will have a look ASAP. Thanks for reporting.

**Comment by jginternational** [comment]
Fixed in #460
